pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -2354,6 +2354,8 @@ var _ = Describe("Manager helper functions", func() {
 			Expect(capacity).ToNot(Equal(nil))
 
 			expectedSize := int64((1 - fakePercentFloat) * float64(*capacity))
+			// The size is expected to be 4k-aligned
+			expectedSize = expectedSize - expectedSize%4096
```

Comment:
Might want to align on 1Mi or 4Mi since there are storage devices out there with larger block sizes (Azure blob comes to mind)

Reply:
1Mi aligned is also 4k aligned, so we are good here.